src/features/Dashboard/components/RecentTransactionList.tsx

```diff
@@ -164,12 +186,15 @@ const makeTxIcon = (type: ITxHistoryType, asset: Asset) => {
 export default function RecentTransactionList({ accountsList, className = '' }: Props) {
   const { getAssetRate } = useRates();
   const { settings } = useSettings();
+  const txTypeMetas = useSelector(getTxTypeMetas);
   const txHistory = useSelector(getMergedTxHistory);
   const { isMobile } = useScreenSize();
 
-  const accountTxs = txHistory.filter((tx) =>
-    accountsList.some((a) => isSameAddress(a.address, tx.to) || isSameAddress(a.address, tx.from))
-  );
+  const accountTxs = txHistory
```

Comment:
Can this be memoized?

Reply:
Could memoize it and add triggers to update when it needs to be updated, but im not sure it's really necessary.

Reply:
Memoizing should be a small change though, you can just use React's `useMemo`, e.g.
```ts
useMemo(() => txHistory.filter(...).map(...), [txHistory.length, accountsList.length])
```

Reply:
I did `[txHistory, accountsList.length]` because txHistory gets updated but not expanded/contracted when `PENDING` transactions change to `COMPLETED`